components/legacy_sidebar/header/sidebar_header_dropdown_button.jsx

```diff
@@ -79,6 +79,7 @@ export default class SidebarHeaderDropdownButton extends React.PureComponent {
                                 verticalAlign: 'top',
                                 marginLeft: 4,
                             }}
+                            openModalOnClick={true}
```

Comment:
Use the aforementioned suggestions.

Define the implementation to open custom status modal and pass a generic `onClick` method here calling that function.